commitlint.config.ts

```diff
@@ -177,6 +204,28 @@ module.exports = {
                     ];
                 },
 
+                'commit-hash-alone': ({raw}: {raw:any}) => {
+                    let rawStr = convertAnyToString(raw, "raw");
+
+                    let offence = false;
+
+                    let urls = findUrls(rawStr)
+
+                    if (urls !== null) {
+                        for (let url of urls.entries()) {
+                            if (isCommitUrl(url.toString())) {
```

Comment:
@realmarv there's an issue here, we need to detect if the URL belongs to the same repo (if it doesn't, it's fair enough that it is a URL); for this, we would check either git remotes or GitHubAcrtions environment variables

Reply:
Done. Please check [this PR](https://github.com/nblockchain/conventions/pull/23)